modules/network/vpc/variables.tf

```diff
@@ -65,37 +70,57 @@ variable "mtu" {
   default     = 0
 }
 
-# the default will create a subnetwork in var.region with the settings noted
-variable "primary_subnetwork" {
+variable "subnetworks" {
   description = <<EOT
```

Comment:
This need to be `<<-EOT` in order to not include the indentation in the multi-line string: 
https://www.terraform.io/language/expressions/strings#indented-heredocs

Reply:
Address on HEAD of PR.